rfcs/0000-rlp-status.md

```diff
@@ -0,0 +1,203 @@
+|             | True   | "PolicyApplied"             | "KuadrantPolicy has been successfully applied                               |
+|             | True   | "PolicyReconciled"          | "KuadrantPolicy has been successfully reconciled"                           |
+|             | False  | "PolicyEnforced"            | "KuadrantPolicy has been successfully enforced"                             |
+|             | False  | "PolicyError"               | "KuadrantPolicy has encountered an error"                                   |
+| Enforced    | True   | "PolicyEnforced"            | "KuadrantPolicy has been successfully enforced"                             |
+|             | False  | "PolicyPartiallyEnforced"   | "KuadrantPolicy has encountered some issues and has been partially applied" |
+|             | False  | "PolicyOverridden"          | "KuadrantPolicy is overridden by [policy-ns/policy-name]"                   |
+| Failed      | True   | "PolicyValidationError"     | "KuadrantPolicy has failed to validate"                                     |
+|             | True   | "PolicyServiceError"        | "KuadrantPolicy has encountered has failed to enforce"                      |
+|             | False  | "PolicyEnforced"            | "KuadrantPolicy has been successfully enforced"                             |
+
+
+## Option 2
+A simpler second option would align more with [GEP-713](https://gateway-api.sigs.k8s.io/geps/gep-713/#conditions). In this case,
+besides the proposed `Accepted` _PolicyType_, the `Enforced` _PolicyType_ would be added to reflect the final state of the policy,
+which means that the policy is showing the synced actual state of the Kuadrant services. The missing `Failed` _PolicyType_
+would be implicitly represented by the `TargetNotFound` and `Invalid` _PolicyTypeReason_.
+
+**Conditions**
+
+All conditions are top-level.
+
+| Type           | Status | Reason              | Message                                                                     |
+|----------------|--------|---------------------|-----------------------------------------------------------------------------|
+| Accepted       | True   | "Accepted"          | "KuadrantPolicy has been accepted"                                          |
+|                | False  | "Conflicted"        | "KuadrantPolicy is conflicted by [policy-ns/policy-name], ..."              |
+|                | False  | "Invalid"           | "KuadrantPolicy is invalid"                                                 |
+|                | False  | "TargetNotFound"    | "KuadrantPolicy target [resource-name] was not found"                       |
+| Enforced       | True   | "Enforced"          | "KuadrantPolicy has been successfully enforced"                             |
+|                | False  | "PartiallyEnforced" | "KuadrantPolicy has encountered some issues and has been partially applied" |
```

Comment:
Not sure I like partially enforced. might be tempted to group that under reason `Unknown` this is similar then to how the gateways themselves reflect transient or uncertain state.

Reply:
Since we can see the information of partially enforced within the `PolicyAffected` type, let's change this one to `Unknown`